src/python/ensembl/io/genomio/seq_region/gbff.py

```diff
@@ -0,0 +1,104 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A SeqRecord wrapper."""
+
+__all__ = [
+    "GBFFRecord",
+]
+
+from dataclasses import dataclass
+import re
+
+from Bio.SeqRecord import SeqRecord
+
+from ensembl.io.genomio.seq_region.exceptions import UnknownMetadata
+
+##############################################
+_MOLECULE_LOCATION = {
+    "apicoplast": "apicoplast_chromosome",
+    "chromosome": "nuclear_chromosome",
+    "kinetoplast": "kinetoplast_chromosome",
+    "linkage group": "linkage_group",
+    "mitochondrion": "mitochondrial_chromosome",
+    "plasmid": "plasmid",
+}
+
+
+@dataclass
+class GBFFRecord:
+    """Wrapper around a SeqRecord object to extract specific data."""
+
+    record: SeqRecord
+
+    def get_genbank_id(self) -> str | None:
+        """Returns the GenBank accession from a given sequence record (if present).
+
+        Only useful for RefSeq sequence records, where the GenBank accession is stored in a comment.
+
+        Args:
+            record: Sequence record.
+
+        """
+        comment = str(self.record.annotations.get("comment", ""))
```

Comment:
Is there any chance the comment field will have something different from a string?
```suggestion
        comment = self.record.annotations.get("comment", "")
```